bin/rialto-parachain/runtime/src/lib.rs

```diff
@@ -302,6 +306,10 @@ parameter_types! {
 	pub const RelayNetwork: NetworkId = NetworkId::Polkadot;
 	pub RelayOrigin: Origin = cumulus_pallet_xcm::Origin::Relay.into();
 	pub UniversalLocation: InteriorMultiLocation = X1(Parachain(ParachainInfo::parachain_id().into()));
+	/// The Millau network ID, associated with Kusama.
+	pub const MillauNetwork: NetworkId = Kusama;
+	/// The RialtoParachain network ID, associated with Westend.
+	pub const ThisNetwork: NetworkId = Westend;
```

Comment:
Sorry, it might be something simple, but I couldn't udnerstand it so far. Why is `MillauNetwork == Kusama` and `ThisNetwork == Westend` ?

Reply:
It's just an association - there's this existing `NetworkId` enum in XCM, but obviously Rialto and Millau are not there. We have also associated Rialto/Millau tokens with Polakdot/Kusama tokens, so this association is kept. For RialtoParachain it's just new association